src-tauri/Cargo.toml

```diff
@@ -1,11 +1,11 @@
 [package]
-name = "app"
+name = "zoo-modeling-app"
```

Comment:
I think this file might be the only place that needs fixing!

Reply:
I'll revert to just this then - not great having so many names flying around but ok